omod-2.0/pom.xml

```diff
@@ -16,7 +16,7 @@
   <properties>
     <openMRSVersion>${openMRS2_0Version}</openMRSVersion>
     <emrapiVersion>1.19</emrapiVersion> 
-    <webservices.restVersion>2.17</webservices.restVersion>
+    <webservices.restVersion>2.21.0</webservices.restVersion>
```

Comment:
I already saw this in the root pom. Do you need to duplicate it here?

Reply:
Resolved thanks